sw/device/lib/crypto/drivers/entropy_kat.c

```diff
@@ -0,0 +1,169 @@
+// Copyright lowRISC contributors.
```

Comment:
nit: I know "kat" stands for "known answer test" but wdyt about renaming this file to `entropy_kat_test.c` for the sake of clarity?

Reply:
I didn't want to use the `_test.c` to avoid giving the impression that it is a functional test. wdyt?